readme.md

```diff
@@ -83,14 +87,16 @@ Configuration:
   --print-config                 Print the normalized config for the given config and options, then exit.  [boolean] [default: false]
   --additional-trace-categories  Additional categories to capture with the trace (comma-delimited).  [string]
   --config-path                  The path to the config JSON.
-                                 An example config file: lighthouse-core/config/lr-desktop-config.js  [string]
+                                              An example config file: lighthouse-core/config/lr-desktop-config.js  [string]
```

Comment:
ha, I did manually edit these ones. I guess `yargs` does something fancy with the whitespace on these too?